### DISCUSSION TITLE
k8s service资源比较多，导致higress-gateway要启动一两个小时

### DISCUSSION THREAD

**Discussion by AbleHou** [alibaba/higress#475] [Q&A]
集群规模比较大，higress-gateway增加副本数时新副本要启动很长时间才能使用，尝试过给容器升配，但收效甚微

**Comment by johnlanni** [comment]
https://github.com/alibaba/higress/pull/433

开启这个配置可以缓解，这个是按需订阅，只推送路由用到的服务，但是要注意路由变更服务时，如果这个路由是该服务的最后一个引用，则该服务会立即被移除，在路由变更到新服务的期间存在一定窗口不可用，这块后续社区还会做优化

**Comment by johnlanni** [comment]
目前如果无法接受这个不可用窗口，也不建议打开该配置。
目前只在master分支开启了这个配置，后续release新版本时会将这个配置关闭，直到这块完成优化后再打开。
